debian/ubuntu-advantage-tools.postinst

```diff
@@ -16,7 +16,9 @@ fi
 
 APT_TRUSTED_KEY_DIR="/etc/apt/trusted.gpg.d"
 
-ESM_INFRA_KEY_TRUSTY="ubuntu-advantage-esm-infra-trusty.gpg"
+# Old key renamed in version 29
+OLD_ESM_INFRA_KEY_TRUSTY="ubuntu-advantage-esm-infra-trusty.gpg"
+ESM_INFRA_KEY="ubuntu-advantage-esm-infra.gpg"
```

Comment:
Should we rename it instead to "ubuntu-pro-esm-infra.gpg" ?

Reply:
didn't add a manual test at first because there is no launchpad bug, but it's trivial to write one - will do.

About renaming it to `ubuntu-pro` - it will break consistency; in this case, should we rename all `ubuntu-advantage` keys to `ubuntu-pro`?

Reply:
I think we should keep consistency for now. Next cycle we should have a roadmap item for "Complete the rename of advantage to pro for all the technical bits" and we can look at renaming these at that point IMO